backend/doc/setting.md

```diff
@@ -473,8 +473,8 @@ To enable online edition on Tracim and allow communication with your edition sof
 
 First you need to enable the edition on the API:
 
-    collaborative_document_edition.activated = True
-    collaborative_document_edition.software = collabora`
+    app_enabled = collaborative_document_edition
```

Comment:
Do you sure about this syntax ?

Reply:
The syntax is incorrect:
- user should **add** an app to existing one, not set only collaborative_document_edition
- **collaborative_document_edition** app need **contents/file** app to work (it's a specific case, most apps are not depend from others)

Reply:
Indeed, I was aware we have to add the app, I thought about ``app_enabled += collaborative_document_edition``.
I wasn't aware of the second point.
I can change for another syntax after a discussion, so we can agree on a solution. Or I can revert this change, since it was a quick fix (I spotted the mistake) and create a new issue to update ``setting.md``.